src/moin/auth/__init__.py

```diff
@@ -3,6 +3,7 @@
 # Copyright: MoinMoin:FrankieChow, MoinMoin:NirSoffer
 # Copyright: 2005-2012 MoinMoin:ThomasWaldmann
 # Copyright: 2007      MoinMoin:JohannesBerg
+# Copyright: 2023      MoinMoin project
```

Comment:
IANAL, but I doubt it works like that.

Reply:
With this entry I just want to document that the module is still maintained. 
An interesting view on copyright notices in open source can be found here:
https://opensource.com/article/20/10/copyright-notices-open-source-software

Reply:
I guess the issue is whether "MoinMoin project" qualifies as an owner under US copyright law and whether that matters in 2023. IANAL either, but my understanding is that a work can be registered in the US using a pseudonym. If you sue someone you have to prove you used the pseudonym. As GIT is tracking all the changes in detail "MoinMoin project" works for me.